providers/supermicro/supermicro.go

```diff
@@ -0,0 +1,343 @@
+package supermicro
```

Comment:
There are a few code comments referencing `redfish` in this file. Might want to be `vendorapi`? Maybe copy/paste left over?

Reply:
oh, right, fixed!